extensions/dropstab_raycast/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [Unreleased]
+### Added
+- Initial release of the `dropstab_raycast` extension.
```

Comment:
## [Initial release] - 2024-08-22
- Initial release of the `dropstab_raycast` extension.